server.js

```
@@ -1022,6 +1022,29 @@ export class FilesCollection extends FilesCollectionCore {
     return mime;
   }
 
+  /*
+   * @locus Anywhere
+   * @memberOf FilesCollection
+   * @name _getUserId
+   * @summary Returns `userId` matching the xmtok token derived from Meteor.server.sessions
+   * @returns {String}
+   */
+  _getUserId(xmtok) {
+    if (!xmtok || !helpers.isObject(Meteor.server.sessions)) return null;
+
+    if (Meteor.server.sessions instanceof Map
+      && Meteor.server.sessions.has(xmtok)
+      && helpers.isObject(Meteor.server.sessions.get(xmtok))
+    ) {
+      return Meteor.server.sessions.get(xmtok).userId
+    } else if (helpers.isObject(Meteor.server.sessions)
+      && xmtok in Meteor.server.sessions
+      && helpers.isObject(Meteor.server.sessions[xmtok])
+    ) {
+      return Meteor.server.sessions[xmtok].userId
+    }
```

Could you please add an `} else {` and throw an Error there to cover this undefined state, too. Even this is highly unlikely it allows us to detect breaking changes on this in future Meteor versions that are related to this issue fast and precise.